src/instance-lookup.js

```diff
@@ -52,7 +73,23 @@ function instanceLookup(server, instanceName, callback, timeout, retries) {
       socket = dgram.createSocket('udp4');
       socket.on('error', onError);
       socket.on('message', onMessage);
-      socket.send(request, 0, request.length, SQL_SERVER_BROWSER_PORT, server);
+
+      if (multiSubnetFailover) {
```

Comment:
> One other suggestion for the instance-lookup case would be to use the same IP address that returned the port, in connectOnPort. I'm not sure if it's mandated but seems safe to do.

Turns out, this is actually the wrong thing to do. What you're doing is correct, it's valid for the IP address that responded with the port to not be hosting a SQL Server instance.